packages/cdk8s-plus-17/package.json

```diff
@@ -70,82 +69,8 @@
     "k8s",
     "kubernetes"
   ],
-  "engines": {
-    "node": ">= 10.17.0"
-  },
```

Comment:
We should specify explicit minNodeVersion in all modules.

Reply:
added